vendor/vendor.json

```diff
@@ -240,8 +245,8 @@
 		{
 			"checksumSHA1": "+bQx8RbUOzmss+tsyqfBvbU4Oi8=",
 			"path": "github.com/keybase/client/go/libkb",
-			"revision": "adfba6190cfe7a03369a53c137837854cdc81d42",
-			"revisionTime": "2018-08-28T18:44:39Z"
+			"revision": "a319ba643ada6c1de524b7906de83ae01a519c38",
```

Comment:
should probably vendor all client/go packages so they're all at the same revision. (need to do it anyway once the client PR goes in, so that the revision is one on master)